src/Makefile.am

```diff
@@ -6,8 +6,111 @@
 
 DIST_SUBDIRS = secp256k1 univalue
 
+# This enables all warnings and treats them as errors. Individual warnings should be
+# 1. fixed,
+# 2. locally silenced with an explanatory comment, or
+# 3. globally silenced here with an explanatory comment.
+#
+# Justifications
+# • `-Wno-aix-compat`: we don’t want to maintain compatibility with IBM AIX compilers
```

Comment:
This is fine; it's different from the options just below because we just don't care about AIX ABI compatibility.